neo/Implementations/Blockchains/LevelDB/LevelDBBlockchain.cs

```diff
@@ -38,7 +38,7 @@ public LevelDBBlockchain(string path)
             Version version;
             Slice value;
             db = DB.Open(path, new Options { CreateIfMissing = true });
-            if (db.TryGet(ReadOptions.Default, SliceBuilder.Begin(DataEntryPrefix.SYS_Version), out value) && Version.TryParse(value.ToString(), out version) && version >= Version.Parse("1.5"))
+            if (db.TryGet(ReadOptions.Default, SliceBuilder.Begin(DataEntryPrefix.SYS_Version), out value) && Version.TryParse(value.ToString(), out version) && version >= Version.Parse("2.6.0"))
             {
```

Comment:
Does this change require all clients to resync the chain?

Reply:
Yes. They can use `chain.acc`.